cmd/otelcol/Dockerfile

```diff
@@ -1,16 +1,20 @@
-FROM alpine:3.12 as certs
+FROM alpine:latest as certs
 RUN apk --update add ca-certificates
 
-FROM alpine:3.12 AS otelcol
+FROM alpine:latest AS otelcol
 COPY otelcol /
 # Note that this shouldn't be necessary, but in some cases the file seems to be
 # copied with the execute bit lost (see #1317)
 RUN chmod 755 /otelcol
 
 FROM scratch
+
+ARG USER_UID=10001
```

Comment:
https://docs.docker.com/engine/reference/builder/#user says

> When the user doesn’t have a primary group then the image (or the next instructions) will be run with the root group.

So could this end up running as root group unless 10001 is a user that happens to exist and has a primary group? I'm not sure how this works really.

Reply:
@jrcamp are you ok with this change?